### DISCUSSION TITLE
When exiting a modal dialog, set to the element that last had focus.

### DISCUSSION THREAD

**Discussion by hgfrey** [primefaces/community#104] [PrimeFaces] [answered]
Is there support in PrimeFaces to set the focus to the (input) element that last had focus when closing a modal dialog?

**Comment by melloware** [comment, marked as answer]
Yes that is now the default in 13.0.0: https://github.com/primefaces/primefaces/issues/10270

it focuses back on the element that opened the dialog.

**Reply by hgfrey** [reply]
That is good news. Thank you...
